neo/test/rawiotest/test_cedrawio.py

```diff
@@ -0,0 +1,19 @@
+import unittest
+
+from neo.rawio.cedrawio import CedRawIO
+
+from neo.test.rawiotest.common_rawio_test import BaseTestRawIO
+
+
+class TestCedRawIO(BaseTestRawIO, unittest.TestCase, ):
+    rawioclass = CedRawIO
+    entities_to_test = [
+        'spike2/m365_1sec.smrx'
+    ]
+    entities_to_download = [
+        'spike2/m365_1sec.smrx'
```

Comment:
Shouldn't this rather be the complete folder of test files?
```suggestion
        'spike2'
```